src/domain/events/__tests__/EventsPage.test.tsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { MockedResponse } from '@apollo/react-testing';
+import { advanceTo } from 'jest-date-mock';
+import React from 'react';
+import wait from 'waait';
+
+import {
+  Event,
+  EventsDocument,
+  MyProfileDocument,
+} from '../../../generated/graphql';
+import {
+  fakeEvents,
+  fakeLocalizedObject,
+  fakeOccurrences,
+  fakeOrganisations,
+  fakePerson,
+  fakePEvent,
+} from '../../../utils/mockDataUtils';
+import { act, render, screen, waitFor } from '../../../utils/testUtils';
+import * as organisationSelectors from '../../organisation/selector';
+import { EVENT_SORT_KEYS, PAGE_SIZE } from '../constants';
+import EventsPage from '../EventsPage';
+
+const eventOverrides: Partial<Event>[] = [
+  { name: fakeLocalizedObject('Tapahtuma 1') },
+  { name: fakeLocalizedObject('Tapahtuma 2') },
+  { name: fakeLocalizedObject('Tapahtuma 3') },
+  { name: fakeLocalizedObject('Tapahtuma 4') },
+  {
+    name: fakeLocalizedObject('Tapahtuma 5'),
+    pEvent: {
+      ...fakePEvent(),
+      // this override mock won't appear in the test atm
+      // TODO: investigate
```

Comment:
@jorilindell If you have time, look if you see what I'm doing wrong here :) at the same time you will learn how these fake functions work :)

Reply:
the fakeOccurrence with startTime of '2020-04-03T09:00:00+00:00' won't appear in the EventsPage when running tests, only the hardcoded ocurrence times

Reply:
Hmm, at least  `console.log(eventsMock.data[4].pEvent.occurrences.edges);` shows
```
[
      {
        cursor: '',
        node: {
          id: '09e8f927-0cde-40c7-9e68-5cf364b7bcd3',
          pEvent: [Object],
          amountOfSeats: 30,
          minGroupSize: 10,
          maxGroupSize: 20,
          autoAcceptance: true,
          languages: [Array],
          startTime: '2020-04-03T09:00:00+00:00',
          endTime: '2020-04-03T09:30:00+00:00',
          placeId: '',
          seatsTaken: 0,
          contactPersons: [],
          createdAt: '',
          enrolments: [],
          studyGroups: [],
          updatedAt: '',
          remainingSeats: null,
          __typename: 'OccurrenceNode'
        },
        __typename: 'OccurrenceNodeEdge'
      }
    ]
```

Reply:
yeah the strange this is that it seems to generate correct mock but can't see that in the component